### DISCUSSION TITLE
Feature Request: Add TouchID support for MacOS

### DISCUSSION THREAD

**Discussion by Larrry** [keepassxreboot/keepassxc#8912] [Ideas]
It would be extremely handy when opening database  )

**Comment by michaelk83** [comment]
It's already supported. https://keepassxc.org/docs/KeePassXC_UserGuide.html#_quick_unlock

**Reply by Larrry** [reply]
I'm little embarrassed because I'm using KeePassXC for many years and never knew that Touch ID support is already here.
I use MacBook Pro 16' 2019 with latest versions of Mac OS (13.0.1) and Keepassx (2.7.4).
There is no TouchID-related options on database unlock screen for me (
The only information I've found - this old issue: https://github.com/keepassxreboot/keepassxc/issues/4818
Is the information mentioned there about checkbox still actual? My unlock screen looks like this: https://external-preview.redd.it/i51uLn0-Mat8R22qMxSg4c4tWKhQA-ibW31IOJEirq0.jpg?auto=webp&s=77bcb91aff95ef076f4fcb9b5bbce00cd094c520

**Reply by droidmonkey** [reply]
Go to settings and then security, make sure you see "Enabled quick unlock" at the top and it's checked. If you don't see that then your Mac may not have touchid enabled.

**Reply by Larrry** [reply]
Still can't find solution for this, there is no option for Touch ID on unlock screen :(
- Touch ID is enabled and work prefectly in other apps (Safari, Pages)
- "Database quick unlock" is enabled (tried to disable/restart/re-enable that for test)
- I've made some random : rebooting, [restart]([TouchID option not displayed](https://apple.stackexchange.com/questions/275968/touchid-option-not-displayed-stops-working)) touch services, checked Content & Privacy settings etc.

Would you share your thoughts on that issue? What can I try next?

**Reply by droidmonkey** [reply]
If the setting is enabled in the security section then you should see the touch id request after locking your database. It will reset if you close your database or restart your computer.
